openfisca_canada_mvohwr/variables/motor_vehicle_operator_hours_of_work_regulation/mvo_driver_classification.py

```diff
@@ -0,0 +1,126 @@
+from openfisca_core.model_api import *
+from openfisca_canada_mvohwr.entities import Person
+
+class mvo_driver_classification_type(Enum):
+    city_motor_vehicle_operator = u'City Motor Vehicle Operator'
+    highway_motor_vehicle_operator = u'Highway Motor Vehicle Operator'
+    bus_operator = u'Bus Operator'
+    other = u'Other'
+
+class mvo__driver_classification(Variable):
+    value_type = Enum
+    possible_values = mvo_driver_classification_type
+    default_value = mvo_driver_classification_type.other
+    entity = Person
+    label = u""
+    definition_period = DAY
+    reference = u"TODO"
+
+    def formula(persons, period, parameters):
+        return select (
+        [
+            persons("mvo__is_cmvo",period),
+            persons("mvo__is_hmvo",period),
+            persons("mvo__is_bus_operator",period)
+        ],
+        [
+            mvo_driver_classification_type.cmvo,
+            mvo_driver_classification_type.hmvo,
+            mvo_driver_classification_type.bus_operator,
+        ],
+        mvo_driver_classification_type.other
+        )
```

Comment:
This is redundant as you can just ask the questions "mvo__is_cmvo", "mvo__is_hmvo" or "mvo__is_bus_operator" when classification is required

Reply:
That makes sense. I'll make a separate variable called "is_other"

Reply:
my thinking was that if its ever used in any type of dictionary context. e.g if we wanted to lookup a value for cmvo then it would be useful to have a string key "cmvo" or "hmvo" etc. (and even better to make it an enum), but maybe that's not common with openfisca